packages/opentelemetry-api/src/trace/NoopTracer.ts

```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright 2019, OpenTelemetry Authors
+ * Copyright 2020, OpenTelemetry Authors
```

Comment:
I think no need to update year when you modify the file.

Reply:
this is automatically done by the `lerna run fix`. I think because the file is a "new" file?